src/mcp/README.md

```diff
@@ -12,6 +12,14 @@ as an argument to operate against the project of your working directory.
 For clients that don't operate within a specific workspace, the Firebase MCP
 server makes tools available to read and write a project directory.
 
+
+### Enable Firebase CLI experiments
+
+Firebase MCP server command aren't visible unless you enable the experiements.
+```
+npx -y firebase-tools experiments:enable mcp
+```
+
```

Comment:
I'm planning to remove the need to enable the experiment before I/O -- that's why it has the `experimental` name in the command itself.

Reply:
When we do that, we can strip `experiments` in instructions & cmd.

Trying to make it easier for folks to test it out right now. Every time, I try to send folks this cmd. 😄